examples/management_canister/src/index.ts

```diff
@@ -199,3 +355,39 @@ export default Canister({
         return state.createdCanisterId;
     })
 });
+
+async function callManagementCanisterasdf(
+    func: string,
+    args?: any,
+    cycles?: bigint
+) {
+    await fetch(`icp://aaaaa-aa/${func}`, {
+        body: serialize({ candidPath: '/candid/management.did', args, cycles })
+    });
+}
+
+async function createCanister() {
+    if (process.env.AZLE_TEST_FETCH === 'true') {
+        const response = await fetch(`icp://aaaaa-aa/create_canister`, {
+            body: serialize({
+                candidPath: '/candid/management.did',
+                args: [{ settings: [], sender_canister_version: [] }],
+                cycles: 50_000_000_000_000n
+            })
+        });
+        return await response.json();
+    } else {
+        return await ic.call(managementCanister.create_canister, {
+            args: [{ settings: None, sender_canister_version: None }],
+            cycles: 50_000_000_000_000n
+        });
+    }
+}
+
+function convertOpt<T>(opt: Opt<T>): [T] | [] {
```

Comment:
Can you make sure this is the exact same code and name as the other place this is used?